src/components/Games/GameItem.tsx

```diff
@@ -0,0 +1,133 @@
+import {StyleSheet, Text, View, Pressable} from 'react-native';
+import React from 'react';
+import * as Progress from 'react-native-progress';
+import {DARKCOLORS} from '../../constants/colors';
+import {FONTFAMILY} from '../../constants/fonts';
+import {FONTSIZE} from '../../constants/sizes';
+import {BlurView} from '@react-native-community/blur';
+import {useNavigation} from '@react-navigation/native';
+import {RootStackScreenProps} from '../../types/navigation/types';
+
+interface Chord {
+  name: string;
+}
+
+interface GameItemProps {
+  status: boolean;
+  score: number;
+  title: string;
+  chordList: Chord[];
+}
+
+const GameItem: React.FC<GameItemProps> = props => {
+  const navigation =
+    useNavigation<RootStackScreenProps<'ChordGame'>['navigation']>();
+  const handleNavigater = () => {
+    {props.status ? navigation.navigate('GameDetail') : null}
+  };
+  return (
+    <Pressable onPress={() => handleNavigater()} style={styles.container}>
+      <Progress.Circle
+        borderWidth={1}
+        thickness={12}
+        color={DARKCOLORS.hightLightColor}
+        strokeCap="round"
+        animated={true}
+        showsText={true}
+        size={120}
+        progress={props.score}
+      />
+      <View style={styles.containerRight}>
+        <Text style={styles.title}>{props.title}</Text>
+        <View style={styles.chordList}>
+          {props.chordList.map((item: any, index: number) => (
+            <Text key={index} style={styles.chord}>
+              {item.name}
+            </Text>
+          ))}
+        </View>
+      </View>
+      {props.status ? null : (
+        <BlurView style={styles.containerUnlock}>
+          <View style={styles.containerBtn}>
+            <Pressable style={styles.btnUnLock}>
+              <Text style={styles.text}>UnLock</Text>
+            </Pressable>
+          </View>
+        </BlurView>
+      )}
+    </Pressable>
+  );
+};
+
+export default GameItem;
+
+const styles = StyleSheet.create({
```

Comment:
You need create a file GameItem.styles.ts and move styles to file GameItem.styles.ts